### PR TITLE
Extra styling to solve all use cases in basket

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
While work was being done to introduce the basket into a project, we noticed that the basket styling doesn't cater for all use cases.
 - for example, if no title is specified then a big gap is left
screen shot 2015-11-24 at 12 01 54

- If there is no [remove] added then the description falls under the price which looks odd
screen shot 2015-11-24 at 12 04 13
This PR aims to solve these issues.

#### What tests does this PR have?
Added 2 tests to check that title is present to display title, and title is present to display price.

#### How can this be tested?
- Look at http://www.ui-toolkit.com and sompare it to this branch (npm run docs).
- Play with the code editor reoving titles and adding long descriptions.
- Try it on mobile and desktop.

#### Screenshots / Screencast
screen shot 2015-11-24 at 11 40 07

#### What gif best describes how you feel about this work?
TROLLEY

---

- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [ ] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by robhuzzey** [comment]
Thanks.... :+1:

File: docs/src/Components.jsx [hunk 1]
```diff
@@ -212,11 +212,11 @@ var Components = React.createClass({
 
           <article id="basket-item">
             <h3>Basket Item</h3>
-            <p>You can use the Basket Item as a way of representing anything with a price next to it. This means you could have a simple description (line of text) or even more complex markup passed in as the child.</p>
+            <p>You can use the Basket Item as a way of representing anything in a basket with or without a price next to it. This means you could have a simple description (line of text) or even more complex markup passed in as the child. Note to have a price, you must declare a title for the product. If the product has no title then description is used.</p>
```
Comment (lukehansell) [inline docs/src/Components.jsx]: seems a strange requirement? What's the background on this?

**Comment by cameronviner** [comment]
Looks good to me :+1:

File: docs/src/Components.jsx (hunk 1 above)
Comment (jodiedoubleday) [inline docs/src/Components.jsx]: So originally something in a basket would always have a name and price. Hoever we looked at existing basket Items in out HX brand and sometimes something in a basket needs to be laid out like a basket item but is not a sellable product, therefore has no price.
E.G. Transfers...
screen shot 2015-11-24 at 15 59 36

We also deduced that an item can't have a price without a title. It just wouldn't make sense to the user.

**Comment by lukehansell** [comment]
+1

**Pull request merged**